aexpect/client.py

```diff
@@ -1769,7 +1770,8 @@ def run_bg(
     encoding=None,
 ):
     """
-    Run a subprocess in the background and collect its output and exit status.
+    Runs the command, awaits for it to finish up to timeout and returns the
+    Expect object (alive or dead, allows interaction).
```

Comment:
Same as above - of course nice clarification too, perhaps we should take it out of the brackets as it is not secondary clarification but a major piece of information to separate the two functions.

Reply:
ok, not sure how to formulate that better without adding long sentences, so let's just go with the brackets form for now...